client/src/components/UserAvatar/styles.css

```diff
@@ -0,0 +1,18 @@
+.user-avatar-container .user-avatar-spinner {
+    line-height: 100px;
+    margin-top: 30px;
+    text-align: center;
+    vertical-align: middle;
+}
+
+.user-avatar-container .avatar-picture {
+    text-align: center;
+    margin-top: 30px;
+    margin-bottom: 20px;
+}
+
+.user-avatar-container .avatar-picture img {
+    width: 200px;
```

Comment:
To make image responsive on small screen
```
max-width: 200px;
width: 100%;
```

Reply:
Good idea! Thank you! 👍